resource-management-client.iml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think this file shouldn't be commited.

Reply:
true. I will remove it.